### ISSUE TITLE
Errors while parsing not yet aired anime

### ISSUE THREAD

**Issue by rr-** [rachmadaniHaryono/python-mal#6]
The library has problems while parsing media with `N/A` in sidebar such as [this one](https://myanimelist.net/anime/32995/Yuri_on_Ice).

``` console
(_ve) rr-@cyclone:~/src/mal$ cat test
#!/bin/env python3
from myanimelist.session import Session
s = Session()
print(s.anime(32995).title)

(_ve) rr-@cyclone:~/src/mal$ ./test
Traceback (most recent call last):
  File "/home/rr-/src/mal/_ve/src/master/myanimelist/media.py", line 263, in parse_sidebar
    media_info['score'] = (decimal.Decimal(score_point), num_users)
  File "/usr/lib64/python3.5/_pydecimal.py", line 597, in __new__
    "Invalid literal for Decimal: %r" % value)
  File "/usr/lib64/python3.5/_pydecimal.py", line 4032, in _raise_error
    raise error(explanation)
decimal.InvalidOperation: Invalid literal for Decimal: 'N/A'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./test", line 4, in <module>
    print(s.anime(32995).title)
  File "/home/rr-/src/mal/_ve/src/master/myanimelist/base.py", line 86, in _decorator
    getattr(self, func_name)()
  File "/home/rr-/src/mal/_ve/src/master/myanimelist/media.py", line 672, in load
    self.set(self.parse(utilities.get_clean_dom(media_page), media_page_original))
  File "/home/rr-/src/mal/_ve/src/master/myanimelist/media.py", line 412, in parse
    media_info = self.parse_sidebar(media_page, media_page_original)
  File "/home/rr-/src/mal/_ve/src/master/myanimelist/anime.py", line 115, in parse_sidebar
    anime_info = super(Anime, self).parse_sidebar(anime_page, anime_page_original)
  File "/home/rr-/src/mal/_ve/src/master/myanimelist/media.py", line 266, in parse_sidebar
    media_info['score'] = (decimal.Decimal(score_tag.text), num_users)
  File "/usr/lib64/python3.5/_pydecimal.py", line 597, in __new__
    "Invalid literal for Decimal: %r" % value)
  File "/usr/lib64/python3.5/_pydecimal.py", line 4032, in _raise_error
    raise error(explanation)
decimal.InvalidOperation: Invalid literal for Decimal: 'N/A'
```

**Comment by rachmadaniHaryono** [comment]
thank for the bug report. the first fix is complete on 19a7e32f0382c5b3f13d0294e83610d688995b30.

can you test it again and check if there is any other error.

**Comment by rr-** [comment]
It works, thanks :3
